docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -0,0 +1,64 @@
+= Parameters
+
+The parent key for all of the following parameters is `metallb`.
+
+
+== `namespace`
+
+[horizontal]
+type:: string
+default:: `syn-metallb`
+
+
+
+== `speaker.secretkey`
+
+[horizontal]
+type:: string
+default:: Vault reference
+
+A binary secret key encoded base64.
```

Comment:
Probably needs clarification. I would like to see the actual default value (something along the lines of `?{clusters/kv/${customer:name}/${cluster:name}/metallb/...}`) shown as the default.

Addtionally I would change the text to:

```suggestion
A vault reference pointing to the Vault secret containing the secret key.
If the key isn't provided as a vault reference, the secret key must be provided in its base64 encoded form.
```